dart/dynamics/UniversalJoint.h

```diff
@@ -54,6 +54,9 @@ class UniversalJoint : public MultiDofJoint<2>
   UniversalJoint(const Eigen::Vector3d& _axis0 = Eigen::Vector3d::UnitX(),
                  const Eigen::Vector3d& _axis1 = Eigen::Vector3d::UnitY(),
                  const std::string& _name = "Universal joint");
+  // TODO(MXG): We should be more consistent about indexing starting from 0 or 1
+  // In the constructor arguments, we start with axis0, but in the member
+  // functions we start with axis1.
```

Comment:
I'm fine with that, although it will require an API change, since the function names themselves are indexing starting from 1. If an API change like that is okay right now, then I can make that change.